gormigrate.go

```diff
@@ -227,10 +227,12 @@ func (g *Gormigrate) RollbackLast() error {
 		return ErrNoMigrationDefined
 	}
 
+	g.begin()
 	lastRunMigration, err := g.getLastRunMigration()
 	if err != nil {
 		return err
 	}
+	g.rollback()
```

Comment:
Are you sure you meant to rollback here?

IMO you should remove this and `return g.commit()` on the last line of this function.

Reply:
Agree, something wrong in my test when this is correct, will take a look at it when I get a bit more time, thanks for looking at this ;)